stimela/backends/flavours/casa.py

```diff
@@ -72,7 +78,7 @@ def get_arguments(self, cab: Cab, params: Dict[str, Any], subst: Dict[str, Any],
         # this works for both python 2.7 and 3.x
         code = f"{command}(**{pass_params})"
 
-        args =  casa.strip().split() + list(casa_opts) + ["-c", code]
+        args =  wrapper + casa.strip().split() + list(casa_opts) + ["-c", code]
```

Comment:
Ooops. @tmolteno could you please try ``wrapper.split()`` here?

Reply:
```
        ├── AttributeError: 'list' object has no attribute 'split'                                                                                                                                      
        └── Traceback:                                                                                                                                                                                  
            ├──   File "/home/tim/github/stimela/stimela/commands/run.py", line 484, in run                                                                                                             
            │       outputs = outer_step.run(is_outer_step=True, subst=subst)                                                                                                                           
            │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                           
            ├──   File "/home/tim/github/stimela/stimela/kitchen/step.py", line 592, in run                                                                                                             
            │       self.cargo._run(params, subst, backend=backend)                                                                                                                                     
            ├──   File "/home/tim/github/stimela/stimela/kitchen/recipe.py", line 1296, in _run                                                                                                         
            │       _, _, _, outputs, _, _ = self._iterate_loop_worker(*args, raise_exc=True)                                                                                                           
            │                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                            
            ├──   File "/home/tim/github/stimela/stimela/kitchen/recipe.py", line 1129, in _iterate_loop_worker                                                                                         
            │       step_params = step.run(backend=backend_settings, subst=subst.copy(), parent_log=self.log)  # make a copy of the subst dict since recipe might modify                                
            │                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                           
            ├──   File "/home/tim/github/stimela/stimela/kitchen/step.py", line 594, in run                                                                                                             
            │       cabstat = backend_runner.run(self.cargo, params=params, log=self.log, subst=subst, fqname=self.fqname)                                                                              
            │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                              
            ├──   File "/home/tim/github/stimela/stimela/backends/runner.py", line 30, in run                                                                                                           
            │       return self.backend.run(cab, params, fqname=fqname, backend=self.opts, log=log, subst=subst,                                                                                        
            │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                         
            ├──   File "/home/tim/github/stimela/stimela/backends/singularity.py", line 270, in run                                                                                                     
            │       args += cab.flavour.get_arguments(cab, params, subst, check_executable=False)                                                                                                       
            │               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                       
            └──   File "/home/tim/github/stimela/stimela/backends/flavours/casa.py", line 81, in get_arguments                                                                                          
                    args = wrapper.split() + casa.strip().split() + list(casa_opts) + ["-c", code]                                                                                                      
```

Reply:
Gah, sorry, remote-control debugging is not always straightforward! See suggestion above rather.